### DISCUSSION TITLE
"embedded" help for functions

### DISCUSSION THREAD

**Discussion by phedoff** [nesbox/TIC-80#1424] [General]
Don't know if it has been asked already - if so: sorry, I couldn't find it - , but:
could you add some kind of help for the functions to the command line? Like
&gt;help _font_ 
which would print the parameters then (text, x, y, [trans...)? I know there aren't so many functions and I could simply write `font()` without any parameter and run the script to get the parameters printed out in the error log, but...

Kind regards

**Comment by joshgoebel** [comment]
This has certainly been asked before, not sure if there is an issue...

**Comment by phedoff** [comment]
Okay, found the now old request [here](https://github.com/nesbox/TIC-80/projects/2#card-48255870). Could someone please close/delete this thread?

**Comment by joshgoebel** [comment]
I don't think you can close discussions. :)

**Reply by phedoff** [reply]
Okay, then.. um.. please nobody read this discussion! ^^
